platform-tests/src/test/java/org/junit/platform/console/ConsoleLauncherIntegrationTests.java

```diff
@@ -92,4 +98,32 @@ void executeScanModules(final String line) {
 		assertEquals(0, new ConsoleLauncherWrapper().execute(args1).getTestsFoundCount());
 	}
 
+	@ParameterizedTest
+	@ValueSource(strings = { "--redirect-stdout", "--redirect-stderr" })
+	void executeWithRedirectedStdStream(String redirectedStream, @TempDir Path tempDir) throws IOException {
+		Path outputFile = tempDir.resolve("output.txt");
+		var line = String.format(
+			"execute -e junit-jupiter --select-method org.junit.platform.console.options.StdStreamTestCase#printTest "
+					+ "%s %s",
+			redirectedStream, outputFile);
```

Comment:
In the future one might stumble upon `StdStreamTestCase` and wonder where it's used so a reference would be helpful.

```suggestion
		var line = String.format(
			"execute -e junit-jupiter --select-class %s %s %s",
			StdStreamTestCase.class.getName(), redirectedStream, outputFile);
```